src/SearchResults/index.js

```diff
@@ -733,30 +810,41 @@ SearchResults.prototype.getFacetValues = function(attribute, opts) {
     return undefined;
   }
 
-  var options = defaultsPure({}, opts, {sortBy: SearchResults.DEFAULT_SORT});
+  var options = defaultsPure({}, opts, {
+    sortBy: SearchResults.DEFAULT_SORT,
+    // if no sortBy is given, attempt to sort based on facetOrdering
+    // if it is given, we still allow to sort via facet ordering first
+    facetOrdering: !(opts && opts.sortBy)
```

Comment:
Doesn't this line override the provided option `{ facetOrdering: true }`? (Flow is hard to understand in the helper so I prefer asking.)

Reply:
it overrides right-to-left, so this is like setting defaults. Not sure why, but this option is like the reverse of object.assign

Reply:
Right, I thought that one left to right.